### ISSUE TITLE
Plucking by property doesn't work after a `rel_length`

### ISSUE THREAD

**Issue by cheerfulstoic** [neo4jrb/neo4j#1072]
This works:

```ruby
@trace_points.pluck(:method_id)
```

This doesn`t:

```
[13] pry(#<MethodsController>)> @trace_points.children(rel_length: 1..10).pluck(:method_id)
 TracePointEntry#children 66ms MATCH (node2:`TracePoint`) WHERE (node2.defined_class = {node2_defined_class}) AND (node2.method_id = {node2_method_id}) MATCH node2<-[rel1:`HAS_PARENT`*1..10]-(result_children) RETURN method_id | {:node2_defined_class=>"Foo", :node2_method_id=>"bar"}
Neo4j::Session::CypherError: method_id not defined (line 1, column 186 (offset: 185))
"MATCH (node2:`TracePoint`) WHERE (node2.defined_class = {node2_defined_class}) AND (node2.method_id = {node2_method_id}) MATCH node2<-[rel1:`HAS_PARENT`*1..10]-(result_children) RETURN method_id"
                                                                                                                                                                                          ^
from /Users/brian/.rvm/gems/ruby-2.2.3/gems/neo4j-core-6.0.1/lib/neo4j-server/cypher_response.rb:225:in `raise_cypher_error'
```

**Comment by cheerfulstoic** [comment]
This was my fault, I gave a `model_class` which wasn't the model but was in fact another Ruby class.  The gem shouldn't be silent in these cases, so I'm going to fix that issue

**Comment by cheerfulstoic** [comment]
@subvertallchris I've made this change and I'm thinking about putting it in to 6.0.x as a patch release.  It would break anybody who is specifying a `model_class` which isn't an `ActiveNode` model.  What do you think about that?

**Comment by subvertallchris** [comment]
I'd consider the gem's silence in that case a bug, so this seems fine!

**Comment by cheerfulstoic** [comment]
Awesome, just released 6.0.3 with this

**Comment by cheerfulstoic** [comment]
Sorry, I meant 6.0.4
